test/fixture/one-pass-one-fail.js

```diff
@@ -1,9 +1,9 @@
 import test from '../../';
 
 test('this is a passing test', t => {
-	t.ok(true);
+	t.truthy(true);
 });
 
 test('this is a failing test', t => {
-	t.ok(false);
+	t.truthy(false);
```

Comment:
I was going to go through and update some of these to a more sensible test, but decided to opt for a direct refactor. I'll go through and update these to be better :+1: